notebooks/config.yaml

```diff
@@ -0,0 +1,21 @@
+# General configurations for this project:
+# seed (int): Seed value for ensuring reproducibility in analyses that involve random operations
+#
+# root_anchor (str): A relative path to a specific file or directory used as a reference
+# (anchor) to determine the project's root directory. Example: If the project root
+# contains a .git/ folder, this can be used to dynamically locate the root directory
+# regardless of the script's location.
```

Comment:
Consider placing these code comments next to the key values they apply to in order to help scale the readability within this file.

Reply:
Some of the documentation is quite lengthy, which can make it challenging to read. I’ll explore the best approach to structure and present these files to improve clarity and accessibility.